docker/docker-compose.yml

```diff
@@ -14,6 +14,11 @@ services:
       - DB_USER=${MONGODB_USER}
       - DB_PASSWORD=${MONGODB_PASSWORD}
       - DB_PORT=${MONGODB_PORT}
+      - SMTP_HOST=${SMTP_HOST}
+      - SMTP_PORT=${SMTP_PORT}
+      - SMTP_USERNAME=${SMTP_USERNAME}
+      - SMTP_PASS=${SMTP_HOST}
+      - PUBLIC_URL=${PUBLIC_URL}
```

Comment:
Can this please be renamed to `PUBLIC_SITE_URL`, so that it is more explicit that it does not refer to the controller service, but rather to the public frontend?